docs/getting_started.rst

```diff
@@ -114,7 +114,25 @@ Suppose we have a transcript in CSV format containing students' basic info and t
 |     202    |     Nick   |   Young   |    Male   |  Physics  |    3.6    |
 +------------+------------+-----------+-----------+-----------+-----------+
 
-Firstly in order to set up a table, we need to specify the schema of this transcript.
+We will not include a header line in this CSV file:
+
+.. code-block:: none
+
+  200,Lucy,Smith,Female,Maths,3.8
+  200,Lucy,Smith,Female,English,3.5
+  201,Bob,King,Male,Maths,3.2
+  202,Nick,Young,Male,Physics,3.6
+
+Instead of using a header line, we will use a separate CSV config JSON file:
+
+.. code-block:: none
```

Comment:
It's a bit not clear on where we store these code blocks. 

I think that we should mention that we store the original data file @ `$WORKING_DIR/data/` and csv config @ `$WORKING_DIR/config/csv-record-reader-config.json`

Same for all code blocks representing the files :)